docs/Metalnx-Architecture.md

```diff
@@ -138,11 +118,11 @@ This is the package where Metalnx exceptions live. If any new exception is neces
 Comparators & Helper classes stay in this package. There are some methods that check if a file is of a certain type, for example `BAM`, `VCF`, `JPEG`, and `Illumina`. There are more like helper methods used by other classes or services.
 
 #### resources
-Under the *resources* folder, we have `xml` files where there is some spring configuration as well as hibernate configuration: DB dialect, username, password, URL and driver name, for example.
+Under the *resources* folder, we have `xml` files where there is some spring configuration.
 		
 ### emc-metalnx-services
 
-This is a Metlanx subproject containing all services that compose our service layer. The service layer is used by our controllers (it will be covered later on) to do a specific job either in iRODS or in the Metalnx database.
+This is a Metlanx subproject containing all services that compose our service layer. The service layer is used by our controllers (it will be covered later on) to do a specific job in iRODS.
```

Comment:
```suggestion
This is a Metalnx subproject containing all services that compose our service layer. The service layer is used by our controllers (it will be covered later on) to do a specific job in iRODS.
```

Reply:
Made the change.